file_paths.md

```diff
@@ -25,13 +25,23 @@ The files annotated by an '*' indicate that they are created by package installa
     /etc/puppetlabs/code *                # :codedir                     contents moved from :confdir
         environments *                    # :environmentpath
           production *
+            hieradata *
             environment.conf *
             manifests *
             modules *
         hiera.yaml *                      # :hiera_config
-        hieradata *                       # n/a
         modules *                         # user modulepath
 
+    /etc/puppetlabs/code-staging          # staging directory            n/a
```

Comment:
What's the thought behind this in /etc? Is it actually configuration? It doesn't feel like it. If feels like it belongs in a content directory under /opt to me. I apologize if I missed that part of the discussion on the thread. There was a lot going on about which implementation before we got down to details (or that's how I read it).